.github/workflows/check_failed_tests.yml

```diff
@@ -41,7 +41,10 @@ env:
 
 jobs:
   check_new_failures:
-    name: " "
+    name: "Find commits for new failing tests"
+    strategy:
+      matrix:
+        run_idx: [1]
```

Comment:
In the future, we might use multiple runners to further reduce the chance of reporting flaky tests